app/coffee/Profiler.coffee

```diff
@@ -0,0 +1,28 @@
+Settings = require('settings-sharelatex')
+logger = require('logger-sharelatex')
+
+deltaMs = (ta, tb) ->
+	return Math.floor(((ta[0]-tb[0])*1e9 + (ta[1]-tb[1]))*1e-6)
```

Comment:
Can we make this a little more readable, perhaps:

```
nanoseconds = (ta[0]-tb[0])*1e9 + (ta[1]-tb[1])
milliseconds = Math.floor(nanoseconds *1e-6)
```

It took me a while to understand as is